mypyc/test-data/run-classes.test

```diff
@@ -273,6 +273,20 @@ class Pokemon(enum.Enum):
 assert Pokemon.magikarp.value == 1
 assert Pokemon.squirtle.name == 'squirtle'
 
+import sys, enum, typing_extensions
+
+# skip test on python 3.8 as it does not allow multiple data-type mixins of the same type
+if sys.version_info[:2] != (3, 8):
+    base_cls: typing_extensions.TypeAlias = enum.IntEnum
+else:
+    base_cls: typing_extensions.TypeAlias = enum.Enum
```

Comment:
FYI, Python 3.8 seemed to break `IntEnum` sometime after Python 3.8.5. It works fine locally with my 3.8.5 installation.

Maybe also worth linking to the BPO?